packages/ui/Layout/Layout.js

```diff
@@ -179,7 +179,27 @@ const LayoutBase = kind({
 		 * This includes support for `align-parts` which is shorthand for combining `align-items`
 		 * and `justify-content` into a single property, separated by a space, in that order.
 		 * This allows you to specify both the horizontal and vertical alignment in one property,
-		 * separated by a space.
+		 * separated by a space. For example, `align="center space-between"` means
```

Comment:
Let's change the line starting from "For example," be sure to make a new line when build docs

Reply:
ok. I will fix it.